pachyderm/templates/etcd/headless-service.yaml

```diff
@@ -1,4 +1,4 @@
----
```

Comment:
I'm going to validate that statefulset works locally so we can eliminate this complexity

Reply:
Done